examples/linear_elasticity/linear_elastic_interactive.py

```diff
@@ -53,7 +54,9 @@ def main():
                                   'vertices in x > %.10f' % (max_x - eps),
                                   'facet')
 
-    field = Field.from_args('fu', nm.float64, 'vector', omega, approx_order=2)
+    field = Field.from_args('fu', nm.float64, 'vector', omega,
+                            space='H1', poly_space_base='lagrange',
```

Comment:
I would remove this line - not a basic tutorial topic, and the other options are not documented anyway.